test/test_achievements_by_person.py

```diff
@@ -1,20 +1,16 @@
 from . import BaseTestCase
 
+
 class TestAchievementsByPerson(BaseTestCase):
-	def test_get_achievements_by_person(self):
-		expected = dict(achievements = [
-							{
-								'age': 21, 
-								'description': 'Founded Apple Computer Company with Wozniak'
-							}, 
-							{
-								'age': 50, 
-								'description': 'Stanford speech'
-							}])
-		response = self.client.get('/achievements/Steve_Jobs/')
-		self.assertItemsEqual(response.json['achievements'], expected['achievements'])
-		self.assertTrue(response.status_code == 200)
+    def test_get_achievements_by_person(self):
+        expected = {'achievements': [{'age': 21,
+                                      'description': 'Founded Apple Computer Company with Wozniak'},
+                                     {'age': 50,
+                                      'description': 'Stanford speech'}]}
+        response = self.client.get('/achievements/Steve_Jobs/')
+        self.assertItemsEqual(response.json['achievements'], expected['achievements'])
```

Comment:
@pepol whatever PEP8 says, I cannot see if it's longer than 80 characters but if it is then sure, please do split it.